usecase/pokemon.go

```diff
@@ -115,3 +117,66 @@ func DeletePokemon(id string) (model.Pokemon, error) {
 
 	return pokemon, err
 }
+
+var pu pokemonUsecaseInterface
+
+func init() {
+	pu = PokemonUsecase{}
+}
+
+/*
+GetPokemonCSV returns a Pokemon list read from a CSV file
+If query params exists, filter the Pokemon to return
+*/
+func GetPokemonCSV(queryParams url.Values) (model.PokemonList, error) {
```

Comment:
Same here as the controller and your services, avoid duplicating code. If you have a structure that implements the methods, it's ok to export them and let them handle the logic. This just makes the file look unnecessarily big

Reply:
👌